src/include/expression/function_expression.h

```diff
@@ -30,7 +30,8 @@ class FunctionExpression : public AbstractExpression {
                      const std::vector<AbstractExpression *> &children)
       : AbstractExpression(ExpressionType::FUNCTION),
         func_name_(func_name),
-        func_(OperatorId::Invalid, nullptr) {
```

Comment:
Does it make sense to separate this into `BuiltinFunctionExpression` and `UDFExpression` ?

Reply:
It would require a bit of re-work on the pg_proc_catalog side to separate the contents of FunctionData https://github.com/cmu-db/peloton/blob/f25ff8ddb452302f178d9a3b62305f95ef4545f7/src/include/catalog/catalog.h#L57 and return the corresponding catalog object. (We would need one for Built-in functions and one for UDF types).

Reply:
We could, but it seems like a lot of work just to separate `function::BuiltInFuncType` and `codegen::CodeContext * `.